---
permalink: /research/
title: "Research"
---

## Ants
Acacia-ants in the genus *Pseudomyrmex* are iconic examples of plant-ant mutualism. These ants obligately nest in the hollow thorns grown by acacias and feed exclusively on the extra-floral nectar and Beltian food bodies produced by their plants. In exchange for this nesting space and food, the ants aggressively protect their hosts against herbivores, pathogenic fungi, and encroaching plants. The ants are incredibly effective and plants without resident ants are quickly overrun by herbivory and disease.

![image-right](/assets/images/pseudo_time_tree_legend-400x400.jpg){: .align-right}

Plant-ant mutualism has evolved at least three times within *Pseudomyrmex*, making this an ideal group in which to study the evolution of this behavior. During my PhD, we compared the genomes of eight species within this genus, including three mutualistic species. Remarkably, these data show that mutualists all have higher rates of molecular evolution genome-wide. These results are now published [here](http://www.nature.com/articles/ncomms12679).

Although we were unable to clearly identify the mechanism underlying these differences in evolutionary rates, this finding piqued my interest in the connections between life history evolution and genome evolution. Questions on this topic form the basis of much of my research program.

Nitrogen isotope ratios for acacia-ants and other genera of ants from the same sites. Acacia-ants feed at a significantly lower trophic level than all other ants sampled.

Ants are the most numerous animals in many terrestrial ecosystems, comprising over 90% of arthropods in the rainforest canopy. This numerical dominance means that ants cannot all be predatory, as was once thought, but must instead feed on more nutrient poor, plant-based resources. Acacia-ants are exemplars of strictly herbivorous ants, living in acacia trees and feeding exclusively on resources provided by these plants. However, the nutritional content of the nectar provided by acacias is insufficient for colony growth and reproduction, leaving open the question of how these ants obtain the necessary nourishment. Many other animals, including humans, are now known to depend on bacteria living in their guts to help them extract nutrients from their food. We, therefore, compared the bacterial communities associated with acacia-ants to related non-mutualists to determine if microbes may help to enrich their diets.
<img src="/assets/images/iso.png" alt="this is a placeholder image" width = "300" height = "322" align = "right">

Instead of a particular bacterial community associated with acacia-ants, we found that community composition is correlated with trophic level, indicating that bacteria likely colonize ant guts based on diet, rather than acacia-ants depending on a specific lineage of coevolved bacteria for nutritional enrichment. Most excitingly, we found that a previously unknown bacterial lineage in the genus *Tokpelaia*, present in both mutualists and non-mutualists, possesses a complete nitrogen recycling pathway, and may contribute to the dietary enrichment of herbivorous ants in general. I am continuing to explore the function of these bacteria and their apparent evolution into obligate symbionts of ants. These findings have been published [here](https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14834).

## Bees

Distribution of dN/dS ratios of free-living *Sodalis* (white circle), sweat bee-associated *Sodalis* (purple), and other insect symbionts (gray). Significance was determined by comparing to the free-living form. The sweat bee-inhabiting *Sodalis* appear to be experiencing widespread relaxed selection.
<img src="/assets/images/sod-phy.png" alt="this is a placeholder image" width = "300" height = "322" align = "right">

I recently found that several strains of bacteria in the genus *Sodalis* have independently colonized sweat bees (Halictidae). In every case, the genomes of these bacteria appear to be experiencing relaxed selection, suggesting a transition to obligate symbiosis. At least some of these strains are shared across bee species and geographic location but appear to compete with each other for territory when found together in the same individual bee. I plan to expand this line of research, what has predisposed *Sodalis* to repeatedly colonizing sweat bees. I will especially focus on the apparent competition between strains as these will likely be informative for understanding how insect-microbial symbioses stabilize over evolutionary time. Our first paper describing these findings is available [here](http://rsos.royalsocietypublishing.org/content/5/7/180369).

I have now received a postdoctoral fellowship from the USDA NIFA program to study the viruses associated with halictids as well. I am using molecular and bioinformatic techniques to characterize the viral communities in sweat bees and other non-model bees that provide important ecosystem services as pollinators.

Phylogeny of halictid genomes being sequenced. Evolutionary relationships are taken from Gibbs et al. (2012) MPE.

In collaboration with my postdoctoral advisor, Dr. Sarah Kocher, I am studying the evolution of social behavior in sweat bees (Halictidae). This clade of bees is remarkably variable in sociality with even closely related species behaving very differently. We are sequencing the genomes of 16 species of these bees in order to identify the convergent genetic changes associated with social evolution.

While genome sequencing has become nearly routine during the past decade, many challenges remain for <em>de novo</em> assembly of non-model genomes. Through the use of linked-read libraries from 10x Genomics, I obtained assemblies with N50 lengths of at least a megabase for every species sequenced. Notably, each species required just $1,500 and a single bee individual to obtain the necessary data. Although these results were already of very high quality, we have also obtained Hi-C chromosome conformation capture data in order to place the resulting scaffolds onto chromosomes, increasing the power and quality of our analyses. I am now exploring the use of Oxford Nanopore sequencing for filling in the few remaining gaps in these assemblies. Using RNAseq data, *ab initio* prediction, and the integration of several gene-finding pipelines, I have also created high-quality gene annotations for these genomes. The integration of coding sequences with population-level genotyping data for each species is beginning to provide insights into the evolution of eusocial behavior.
